src/controllers/controller.js

```diff
@@ -50,13 +50,34 @@ function addDeveloperToProject(project, developer) {
 
 //TODO deleteDeveloperFromProject
 
+async function getAllProjects() {
+  let projectSnapshot = await db.collection('Projects').get();
+  let projectArray = [];
+
+  for (project of projectSnapshot._docs()) {
+    let projectData = project.data();
+    let projectObj = new Project(project.id, projectData.name, new Date(projectData.startDate.toDate()), new Date(projectData.endDate.toDate()), projectData.estimatedHours);
```

Comment:
more consts. Also what's this `toDate()` method?

Reply:
Firebase giver timestamps og ikke dates. Med toDate bliver det converteret til en date